templates/lxc-oci.in

```diff
@@ -36,16 +35,19 @@ done
 LOCALSTATEDIR=@LOCALSTATEDIR@
 LXC_TEMPLATE_CONFIG=@LXCTEMPLATECONFIG@
 LXC_HOOK_DIR=@LXCHOOKDIR@
+ATOMFS="atomfs"
```

Comment:
How about accepting a new template argument --mount-helper, so I could pass along 'composefs' or whatever?

e.g.
```
lxc-create -t oci -n new -- --url docker://zothub.io/tools/busybox:stable --mount-helper /usr/bin/atomfs
```

Reply:
> 

I can do that.